src/main/java/io/github/thebusybiscuit/slimefun4/core/commands/SlimefunTabCompleter.java

```diff
@@ -47,11 +52,11 @@ else if (args[0].equalsIgnoreCase("research")) {
                 return createReturnList(suggestions, args[2]);
             }
             else {
-                return null;
+                return Collections.emptyList();
```

Comment:
Returning an empty list will disable tab completion. Congratulations...
It needs to return null for a reason.

Reply:
That's.. that's the point. It's not supposed to default to the default command executor.